src/jbuild.ml

```diff
@@ -1352,6 +1352,39 @@ module Alias_conf = struct
          })
 end
 
+module Tests = struct
+  type t =
+    { exes    : Executables.t
+    ; locks   : String_with_vars.t list
+    ; package : Package.t option
+    ; deps    : Dep_conf.t list
+    }
+
+  let t =
+    record
+      (Buildable.t >>= fun buildable ->
+       field_oslu "link_flags"                             >>= fun link_flags ->
```

Comment:
It might not be the best place to start a discussion about code style, but I don't think that aligning these `(>>=)` is a good idea. It will either require some maintenance effort every time we modify anything in that block, or become unaligned. This is not worth the minor readability benefit IMO.

Reply:
Agreed. I just try to follow whatever existing style is in place. Here I've failed because it is indeed troublesome without some sort of automation.